quantus/helpers/explanation_func.py

```diff
@@ -1,26 +1,39 @@
 """This modules contains explainer functions which can be used in conjunction with the metrics in the library."""
 from typing import Union
+
+import numpy as np
 import torch
 import scipy
 import random
+from importlib import util
 import cv2
 from captum.attr import *
 import warnings
 from .utils import *
 from .normalise_func import *
 
 
+if not (util.find_spec("captum") or util.find_spec("tf_explain")):
+    raise ImportError(
+        "Explanation library not found. Please install Captum for torch models and tf-explain for TensorFlow."
+    )
+if util.find_spec("captum"):
```

Comment:
make a global flag from this logic

Reply:
but it in __init__ file

Reply:
__EXTRAS__ = True if util.find_spec("captum") or util.find_spec("tf_explain")) else False

Reply:
maybe make that `__EXTRAS__ = ...`

Reply:
`__EXTRAS__ = util.find_spec("captum") or util.find_spec("tf_explain")`


but, one question: is the attribution method availability (which surely is not 100% identical between captum and tf_explain covered, or will the user potentially be surprised by  unavailable attribution methods?